src/main/java/net/minecraftforge/common/capabilities/CapabilityManager.java

```diff
@@ -5,48 +5,60 @@
 
 package net.minecraftforge.common.capabilities;
 
-import net.minecraftforge.fml.ModLoader;
-import net.minecraftforge.forgespi.language.ModFileScanData;
+import net.minecraft.resources.ResourceLocation;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.jetbrains.annotations.NotNull;
 
 import java.util.IdentityHashMap;
-import java.util.List;
 
 import static net.minecraftforge.fml.Logging.CAPABILITIES;
 
+/**
+ * The Capability Manager is responsible for making and 
+ * providing {@link CapabilityType} instances.
+ */
 public enum CapabilityManager
 {
     INSTANCE;
-    static final Logger LOGGER = LogManager.getLogger();
 
+	private static final Logger LOGGER = LogManager.getLogger();
+    private static final IdentityHashMap<ResourceLocation, CapabilityType<?>> PROVIDERS = new IdentityHashMap<>();	
```

Comment:
`IdentityHashMap` seems to be a mistake here.

Reply:
Hm, yeah, not sure how that got there.  Think I got it mixed up with some other map that needs to be identity.